requirements/dev-requirements.yml

```diff
@@ -22,5 +22,6 @@ dependencies:
   - sphinx-autobuild
   - toml>=0.10.2
   - pip
```

Comment:
Not related to your PR, but the "pip" entry at line 13 seems to be a duplicate. Since pip is already listed here maybe we could delete the entry at line 13.